### PR TITLE
Project: Recipe App

### PR DESCRIPTION
**Pull request by Gleennkar**
The recipe app project has two members :
https://github.com/asnganje
https://github.com/Gleennkar

The following task has being implemented on this project as par the requirements
- A login page.
- A registration page.
- A food list (with all CRUD implementation, except for 'update').
- A recipes list (with all CRUD implementation, except for 'update').
- A general shopping list view (all missing food for all your recipes and total price).
- A list of all public recipes from other users with their names and total prices.
- Updated README.md
- Check for linter errors

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
+  has_many :recipes, dependent: :destroy
+  has_many :foods, dependent: :destroy
+end
```
Comment (AnselemOdims) [inline app/models/user.rb]: - [ ] At the moment, when a user signs up they are not redirected to the home page and there is no information telling them that they have registered successfully and need to confirm their email. Please kindly ensure this is fixed to also improve the user experience

File: app/views/recipes/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+          <div>
+            <%= form.submit id: "create_recipe_food_btn" %>
+          </div>
+        </div>
+      </div>
+  <% end %>
+
+  <div class="generate-shopping-list-container" id="pop-up">
+      <%=form_with url: general_shopping_list_index_path, method: :get do |f| %>
+        <div>
```
Comment (AnselemOdims) [inline app/views/recipes/show.html.erb]: - [ ] Please kindly ensure that this form only appears when the "Add ingredient" button is clicked.

image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @team, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please kindly ensure to include the model tests as all the tests are currently empty
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 3]
```diff
@@ -1 +1,121 @@
...
+- To clone the repository run this command:
+
+ `git clone https://github.com/asnganje/Food-Recipe.git`
+
+
+
+- To access the cloned directory run:
+
+  `cd blog_app`
+
```
Comment (tresorsawasawa) [inline README.md:71]: - [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you update the above command to match the correct name of your project as the current is wrong. Good: `cd Food-Recipe`,  Bad: `cd blog_app`

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @Gleennkar and @asnganje 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
